packages/pds/src/api/com/atproto/server/getAccountInviteCodes.ts

```diff
@@ -33,18 +33,25 @@ export default function (server: Server, ctx: AppContext) {
       // note: even if a user is disabled from future invites, we still create the invites for bookkeeping, we just immediately disable them as well
       const now = new Date().toISOString()
       if (createAvailable && ctx.cfg.userInviteInterval !== null) {
+        // for the sake of generating routine interval codes, we do not count explicitly gifted admin codes
+        const routineCodes = userCodes.filter(
+          (code) => code.createdBy !== 'admin',
+        )
+        const unusedRoutineCodes = unusedCodes.filter(
+          (code) => code.createdBy !== 'admin',
+        )
```

Comment:
Not really a suggestion, but I realized we have all the types we need kicking around to write both the query builder parts and js logic in a similar style if it ever proves useful.  An example for a case like we have here:
```ts
const isRoutine = (code: InviteCode) =>
  code.createdBy !== 'admin'

const whereIsRoutine = (qb: Where<'invite_code'>) =>
  qb.where('createdBy', '!=', 'admin')

// type util to make it a bit prettier
type Where<T extends keyof DatabaseSchemaType> = WhereInterface<DatabaseSchemaType, T>
```